rasa_core/constants.py

```diff
@@ -22,6 +22,8 @@
 
 FORM_SCORE = 1.3
 
+MAPPING_SCORE = 1.4
```

Comment:
needs to be merged with https://github.com/RasaHQ/rasa_core/pull/1735 (I guess whoever merges first into master wins :D )

Reply:
yeah I'm about to merge mine but @imLew I can help you adapt yours to fit mine, should just require adding a default priority of 5 to this policy's` __init__` i think, and then using `1.0` instead of `MAPPING_SCORE`